src/output.h

```diff
@@ -328,6 +328,7 @@ int right_print( const catacurses::window &w, int line, int right_indent,
 void insert_table( const catacurses::window &w, int pad, int line, int columns,
                    const nc_color &FG, const std::string &divider, bool r_align,
                    const std::vector<std::string> &data );
+std::string display_satiety_bar( int calpereffv );
```

Comment:
```suggestion
std::string satiety_bar( int calpereffv );
```

I think this would be a better name (see related comment in the `.cpp` file).